clientd3d/music.c

```diff
@@ -27,10 +27,16 @@ static Bool has_midi = False;    /* Can system play MIDI files? */
 
 static Bool playing_midi = False;  /* Is a MIDI file currently playing as an effect? */
 static Bool playing_music = False;  /* Is a MIDI file currently playing as background? */
+static Bool paused_music = False;   /* Is background music paused? */
+static Bool case_bg = False;   /* For loading midi files: which device to load to */
```

Comment:
This is a device?  Then why is it named "case_bg"?

Reply:
This has been fixed.  The case_bg variable was removed completely and the OpenMidiFile() function was refactored.  An argument is used to identify which device to open the Midi File to and this allowed for reduction in the amount of code required in this function.